test/FlashRollover.ts

```diff
@@ -619,7 +621,7 @@ describe("FlashRollover", () => {
             const tx = await flashRollover.connect(borrower).rolloverLoan(contracts, loanId, loanTerms, v, r, s);
             const receipt = await tx.wait();
             const gasUsed = receipt.gasUsed;
-            expect(gasUsed.toString()).to.equal("885296");
+            expect(gasUsed.toString()).to.equal("931867");
```

Comment:
note: not sure currently why gas went up here given we're still just moving NFT tokens around.. looking into it